scripts/SolidityCommentCropper.java

```diff
@@ -0,0 +1,107 @@
+import java.io.*;
+
+public class SolidityCommentCropper {
+    private static final String PRODUCT_COMMENT_START = "@product";
+
+    private static final String[] filesToCrop = {
```

Comment:
closed-list approach, as used here, omits a lot of nonsense files found in the codebase and also prevents duplicate template/sol file messages